can-validate-validatejs.md

```diff
@@ -1,21 +1,71 @@
 @module {Object} can-validate-validatejs
 @parent can-ecosystem
-@group can-validate-validatejs.core 1 can-validate-validatejs Core
 @package ./package.json
 
-@description A plugin for CanJS that wraps any validation library to `can.validate`.
-**Can-Validate doesn't do any validation of its own** but instead provides some abstraction to your library of choice. The chosen library is registered with can-validate using a shim.
+@description Create [can-validate/validator] functions using [validate.js](https://validatejs.org/).
 
-@type {Object}
+@signature makeValidator(constraints)
```

Comment:
The signature should be wrapped in backticks, so it goes from this:
<img width="397" alt="screen shot 2017-01-12 at 12 10 23 pm" src="https://cloud.githubusercontent.com/assets/10070176/21906497/63182d44-d8c0-11e6-8d8e-bdc28e258899.png">

To this:
<img width="485" alt="screen shot 2017-01-12 at 12 12 06 pm" src="https://cloud.githubusercontent.com/assets/10070176/21906510/696d7f46-d8c0-11e6-9fbe-d38d48c207c5.png">